.github/workflows/CI.yml

```diff
@@ -17,61 +28,115 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Set up Python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@v1
+        with:
+          PYTHON_VERSION: ${{ env.PYTHON_VERSION }}
       - name: Environment Information
         run: npx envinfo
       - name: Build
-        run: ./configure && make -j2
+        run: make build-ci -j2 V=1
+      - name: Test
+        run: make run-ci -j2 V=1
   build-windows:
     runs-on: windows-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Environment Information
-        run: npx envinfo
+      - name: Set up Python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@v1
+        with:
+          PYTHON_VERSION: ${{ env.PYTHON_VERSION }}
       - name: Install deps
         run: choco install nasm
+      - name: Environment Information
+        run: npx envinfo
       - name: Build
         run: ./vcbuild.bat
   build-macOS:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Set up Python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@v1
+        with:
+          PYTHON_VERSION: ${{ env.PYTHON_VERSION }}
       - name: Environment Information
         run: npx envinfo
       - name: Build
-        run: ./configure && make -j8
+        run: make build-ci -j8 V=1
+      - name: Test
+        run: make run-ci -j8 V=1
```

Comment:
If we're adding testing, please rename the job from "build-macOS" to "test-macOS" (to mark them as different from the "build-*" jobs that only build and do not run tests).